test/utils/DeploymentSetup.sol

```diff
@@ -151,7 +152,10 @@ abstract contract PreMigrationSetup is Setup {
         compoundStrategy = CompoundStrategy(compoundStrategyProxy);
         compoundStrategy.initialize(VAULT, compoundRewardPool);
         compoundStrategy.setPTokenAddress(USDC, 0x9c4ec768c28520B50860ea7a15bd7213a9fF58bf);
+        compoundStrategy.setPTokenAddress(USDCe, 0xA5EDBDD9646f8dFF606d7448e414884C7d905dCA);
```

Comment:
Please convert this address to a constant

Reply:
Done: https://github.com/Sperax/USDs-v2/commit/ec81a09bc8f6a7b573378066c12ea6100d344a11

Reply:
Perf, thanks